content/kubermatic/main/installation/install-kkp-CE/_index.en.md

```diff
@@ -95,12 +95,15 @@ tar -xzvf kubermatic-ce-v${VERSION}-linux-amd64.tar.gz
 {{% /tab %}}
 {{% tab name="MacOS" %}}
 ```bash
+# Determine your macOS processor architecture type
+# Replace 'amd64' with 'arm64' if using an Apple Silicon (M1) Mac.
+export ARCH=amd64
```

Comment:
should use `uname -m` (or something similar; I _think_ `uname -m` is the way to go but please double-check) to auto-determine the cpu type - and no need to export it.

Reply:
no, you can't use it for `amd64`, it's either `x86_64` or `i386`

Reply:
ah damn. okay, that's unfortunate.